.travis.yml

```diff
@@ -2,9 +2,7 @@ sudo: required
 dist: trusty
 language: ruby
 bundler_args: --without benchmarks development
-# Temporary workaround for broken Rubygems on Travis
```

Comment:
Every so often, a new Rubygems release hoses tests. I want to leave the pin. No problem updating the pinned version though.